standards.md

```diff
@@ -0,0 +1,321 @@
+# Earth System Metrics and Diagnostics Standards
+
+## Author list
+Ana Ordonez, Lawrence Livermore National Laboratory  
+Tom Jackson, NOAA Geophysical Fluid Dynamics Laboratory  
+Paul Ullrich, University of California Davis    
+Aparna Radhakrishnan, Princeton University/NOAA Geophysical Fluid Dynamics Laboratory
+
+
+## Table of Contents
+1. [Best Practices](#bestpractices)
+2. [DEM Framework File Format](#demframework)
+    1. [Settings file](#settingsfile)
+    2. [Contents file](#contentsfile)
+3. [Runtime Environment](#runtimeenvironment)
+4. [Common Output Bundle Format](#commonoutput)
+5. [Observational Data Format](#obsdataformat)
+6. [Observational Data Server](#obsdataserver)
+7. [Common Metric Output Format](#commonmetric)
+    1. [DIMENSIONS](#dimensions)
+    2. [RESULTS](#results)
+
+
+## Best Practices <a name="bestpractices"></a>
+
+### Nomenclature 
+In this document we refer to MDTF process-oriented diagnostics (PODs) and CMEC metric kernels (MKs) as data evaluation modules (DEMs).
+
+### Supported Languages
+Python is the preferred language for development. However, there are certain cases where Python may not be performant or may not have the requisite functionality (i.e. certain statistical analysis routines). Nonetheless, if the kernel evaluation routines are not in python, python interfaces should be provided for accessing the functionality of the kernel.  
+
+If compiled code is used, a robust build system should be provided.  Compiled code should minimally rely on third-party dependencies and/or should be available through a conda package.
+
+### Python version
+Official support (fixing bugs) of python 2.7 will end on January 1, 2020.  
+
+The framework will support Python >= 3.6 (version 3.6 set by the asyncio module) and, as resources permit, version 2.7.  
```

Comment:
This should be changed to >=3.7 for consistency with version specified in the conda env files

Reply:
@wrongkindofdoctor Thanks for the review. Do you know what the reason is for 3.7? It could be helpful to add that information as well.